WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -27,18 +27,18 @@ public static async Task<EstimateSmartFeeResponse> EstimateConservativeSmartFeeA
 		};
 	}
 
-	private static EstimateSmartFeeResponse SimulateRegTestFeeEstimation(int confirmationTarget)
+	private static EstimateSmartFeeResponse SimulateTestNetworksFeeEstimation(int confirmationTarget, int? fixedSatoshiPerByte = null)
```

Comment:
`fixedSatoshiPerByte` is never used. But this should be for regtest only because we never have fees. For testnet it should work exactly as it does for mainnet

Reply:
I was under the wrong assumptions that mempool was not correctly working on testnet4